docs/declarative-customization/column-formatting.md

```diff
@@ -432,33 +432,221 @@ To use the sample below, you must substitute the ID of the Flow you want to run.
 
 ```JSON
 {
-	"$schema": "https://developer.microsoft.com/json-schemas/sp/column-formatting.schema.json",
-	"elmType": "span",
-	"style": {
-		"color": "#0078d7"
-	},
-	"children": [
-    {
-		"elmType": "span",
-		"attributes": {
-			"iconName": "Flow"
-		}
+    "$schema": "https://developer.microsoft.com/json-schemas/sp/column-formatting.schema.json",
+    "elmType": "span",
+    "style": {
+        "color": "#0078d7"
     },
-    {
-		"elmType": "button",
-		"style": {
-			"border": "none",
-			"background-color": "transparent",        
-			"color": "#0078d7",    
-			"cursor": "pointer"
-		},
-		"txtContent": "Send to Manager",
-		"customRowAction": {
-			"action": "executeFlow",
-			"actionParams": "{\"id\": \"183bedd4-6f2b-4264-855c-9dc7617b4dbe\"}"
-		}          
-    }        
-  ]
+    "children": [
+        {
+            "elmType": "span",
+            "attributes": {
+                "iconName": "Flow"
+            }
+        },
+        {
+            "elmType": "button",
+            "style": {
+                "border": "none",
+                "background-color": "transparent",
+                "color": "#0078d7",
+                "cursor": "pointer"
+            },
+            "txtContent": "Send to Manager",
+            "customRowAction": {
+                "action": "executeFlow",
+                "actionParams": "{\"id\": \"183bedd4-6f2b-4264-855c-9dc7617b4dbe\"}"
+            }
+        }
+    ]
+}
+```
+
+## Formatting multi-value fields
+You can use column formatting to apply styles to each member of a multi-value field of type Person, Lookup and Choice.
+
+### Basic text formatting
+The following image shows an example of multi-value field formatting applied to a Person field.
+
+![List of buttons that notifies all assignees of each item, first row is empty, second row reads "Send email to Loyd Barham", third row reads "Send email to all 3 members"](../images/sp-columnformatting-multi-value-0.png)

```

Comment:
@blueset  can you please put an image which does not have border along(Just like in other places) also showing more examples of 'people in a sentence'